src/open_inwoner/openzaak/documents.py

```diff
@@ -87,3 +94,37 @@ def download_document(url: str) -> Optional[Response]:
         logger.exception("exception while making request", exc_info=e)
     else:
         return res
+
+
+def upload_document(
+    file: InMemoryUploadedFile, title: str, user_choice: int, source_organization: str
+) -> dict:
+
+    client = build_client("document")
+    if client is None:
+        return
+
+    document_body = {
+        "bronorganisatie": source_organization,
+        "creatiedatum": date.today().strftime("%Y-%m-%d"),
+        "titel": title,
+        "auteur": "Open Inwoner Platform",
```

Comment:
I would use the name of the logged-in user in OIP for this. OIP is just a vessel, we didn't create the document :)